### PR TITLE
Fix font sizes, whitespace and add copy.

### PR DESCRIPTION
**Pull request by carolinegreen**
## What

Fix font sizes, whitespace, and emphasis on what is bold in the table on the member's page to be closer to Steve's design.  Also add copy describing user roles.

## Before

screen shot 2018-05-22 at 16 35 06

screen shot 2018-05-22 at 16 24 47

## After

screen shot 2018-05-22 at 16 22 44

screen shot 2018-05-22 at 16 22 59

## How to review

Run the application.  Check design against @stephenjoe1's prototype.  Check copy.

## Who can review

Anyone who isn't me.

### CODE REVIEW

File: src/users/users.njk [hunk 1]
```diff
@@ -6,19 +6,14 @@
...
-  {{ super() }}
-  <link rel="stylesheet" type="text/css" href="./users.scss">
-{% endblock %}
-
 {% block main %}
 <div class="govuk-o-grid">
   <div class="govuk-o-grid__item govuk-o-grid__item--two-thirds">
-    <h1 class="heading-large">
+    <h1 class="govuk-heading-l">
       Team members
```
Comment (paroxp) [inline src/users/users.njk]: You took the `Team` away from the button. Do you think it's worth taking away from here too?

Comment (carolinegreen) [reply src/users/users.njk]: I was leaving "team" here as I thought it was nice as a an introduction and we could shorten it to "members" everywhere else as people already had the context that these were "team members" from the heading.  Putting "team" in front of the heading also makes it seem a little bit less repetitive when the paragraph below starts with "members".

Comment (carolinegreen) [reply src/users/users.njk]: Maybe we should have content designer look at it though.

Comment (paroxp) [reply src/users/users.njk]: Maybe, my only worry is that introduction of a new unfamiliar to CF word which may confuse people. What is a team? We already have Organisation Members and Space Members etc.

**Pull request merged**